src/DynamoCoreWpf/Interfaces/IWorkspaceViewModel.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Dynamo.Models;
+using System;
+
+namespace Dynamo.Wpf.Interfaces
+{
+    public interface IWorkspaceViewModel
+    {
+        WorkspaceModel Model { get; }
```

Comment:
I know it's tempting to do this, we should not do this.  We should only expose `IWorkspaceModel` as an API.  

`WorkspaceModel` is not a good API.  :(  It is going to change.  In fact, it is changing right now in fundamental ways.  For example, @lukechurch is totally changing the way we expose nodes, connectors, and groups to third parties.  We must not expose third parties to this kind of chaos. 

Keep in mind that the work you're doing is ostensibly to add this dialog for submitting customizers, but it's also establishing an API for `ViewExtension`'s, which will potentially be consumed by many, many third parties.  We want to keep this API as stable as possible for those third parties.